oe_theme.libraries.yml

```diff
@@ -26,3 +26,10 @@ dialogs:
   dependencies:
     - core/drupal
     - oe_theme/base
+
+gallery:
+  js:
+    js/gallery.js: {}
+  dependencies:
+    - core/drupal
+    - oe_theme/base
```

Comment:
Libraries that are only specific to a pattern should be defined on the pattern definition and not globally. For more info refer to the [UI Patterns documentation](https://ui-patterns.readthedocs.io/en/8.x-1.x/content/patterns-definition.html#expose-pattern-assets-as-libraries).

Reply:
We've fixed autoinit by patch for fixing bug in ecl-twig. Now we don't need to declare aditional library.